boards/nordic/nrf54h20pdk/board.cmake

```
@@ -1,3 +1,15 @@
 # SPDX-License-Identifier: Apache-2.0
 
 include(${ZEPHYR_BASE}/boards/common/nrfjprog.board.cmake)
+
+if(CONFIG_BOARD_NRF54H20PDK_NRF54H20_CPUAPP OR CONFIG_BOARD_NRF54H20PDK_NRF54H20_CPURAD)
+  if(CONFIG_BOARD_NRF54H20PDK_NRF54H20_CPURAD)
+    set(
+      JLINK_TOOL_OPT
+      "-jlinkscriptfile ${CMAKE_CURRENT_LIST_DIR}/support/nrf54h20_cpurad.JLinkScript"
+    )
+  endif()
+
+  board_runner_args(jlink "--device=CORTEX-M33" "--speed=150" "--tool-opt=${JLINK_TOOL_OPT}")
```

I don't think the speed needs to be this low on ASIC
```suggestion
  board_runner_args(jlink "--device=CORTEX-M33" "--speed=4000" "--tool-opt=${JLINK_TOOL_OPT}")
```